normandy/control/static/control/admin/sass/partials/_common.scss

```diff
@@ -134,6 +138,13 @@ thead {
   font-weight: 600;
   font-size: 13px;
   font-family: $OpenSans;
+
+  th {
+    outline: 0;
+    padding: 5px 0px;
+    &:hover { cursor: pointer; }
+    i { color: $secondaryColor; }
+  }
```

Comment:
Small detail, but maybe add something like: 
`&.reactable-header-sort-asc i {color: $green;}
&.reactable-header-sort-desc i {color: $red;}`
To give an indication that sorting is taking place, like this 
<img width="171" alt="screen shot 2016-06-22 at 9 56 45 am" src="https://cloud.githubusercontent.com/assets/12304716/16275638/b5843458-385f-11e6-82ff-4f5d0b422d2a.png">